proptest-state-machine/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "proptest-state-machine"
+version = "0.1.0"
+authors = ["Tomáš Zemanovič"]
+license = "MIT/Apache-2.0"
+edition = "2018"
+repository = "https://github.com/proptest-rs/proptest"
+homepage = "https://proptest-rs.github.io/proptest/proptest/state-machine.html"
+documentation = "https://docs.rs/proptest-state-machine/latest/proptest-state-machine/"
+keywords = ["property", "testing", "quickcheck", "fuzz", "state-machine"]
+categories = ["development-tools::testing"]
+description = """
+State machine based testing support for proptest.
+"""
+
+[dependencies]
+proptest = { version = "1.2.0", path = "../proptest" }
```

Comment:
I think CI may be busted because the latest crate version is 1.1.0 not 1.2.0

Reply:
I separated the needed proptest changes into https://github.com/proptest-rs/proptest/pull/324